lib/request.js

```diff
@@ -73,8 +74,9 @@ req.header = function header(name) {
   var lc = name.toLowerCase();
 
   switch (lc) {
-    case 'referer':
     case 'referrer':
+      deprecate('The "Referrer" header is non-standard, and special-casing for it will be dropped in version 6.');
```

Comment:
Can we change this to read "in future versions".  I don't think we want to commit to when it will be removed, just that we plan on doing it.

Reply:
Good point – I've updated the language.